releases/v1.6.19.toml

```diff
@@ -0,0 +1,20 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.6.18"
+
+pre_release = false
+
+preface = """\
+The nineteenth patch release for containerd 1.6 updates hcsshim tag to v0.9.7
+
+### Notable Updates
+
+* **[release/1.6] go.mod: Bump hcsshim to v0.9.7 ([#8153](https://github.com/containerd/containerd/pull/8153))
```

Comment:
Lets get a description of what was added/fixed in the bump. Something like https://github.com/containerd/containerd/blob/v1.6.18/releases/v1.6.13.toml#L19